Directory.Packages.props

```diff
@@ -15,7 +15,10 @@
 
   <!-- These versions are not updated by dependency flow as they aren't produced live anymore. -->
   <ItemGroup>
+    <PackageVersion Include="runtime.native.System.Data.SqlClient.sni" Version="4.4.0" />
+    <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
     <PackageVersion Include="System.Reflection.Emit" Version="4.7.0" />
+    <PackageVersion Include="System.Text.Encoding.CodePages" Version="8.0.0" />
```

Comment:
@ericstj any opinion on the version used here? Should this be 6.0.0 instead?

Reply:
I'm OK with the latest LTS here.  That said I was surprised to see these since they are only needed for netstandard and I expect they are implementation dependencies.  I saw the property for GeneratePlatformNotSupportedAssemblyMessage but that doesn't seem to be used at all.  If we're truly trying to build a PNSE we could probably remove those.  Need to get clarity on that.

Reply:
I wasn't aware I needed to include them because of the fact that we aren't building PNSE assembly anymore.. if we can get PNSE to build like the old package, we can remove them again.